app/Http/Controllers/CorrespondencesController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Correspondence;
+
+class CorrespondencesController extends Controller
+{
+    public function index(Request $req)
+    {
+        try {
+            $correspondences = Correspondence::paginate();
```

Comment:
```suggestion
            $correspondences = Correspondence::paginate(10);
```
Aqui podemos limitar quantos itens serão paginados, mas é só para lembrar, se não me engano o default é 15.